pkg/codegen/templates/gin/gin-register.tmpl

```diff
@@ -2,6 +2,7 @@
 type GinServerOptions struct {
     BaseURL string
     Middlewares []MiddlewareFunc
+    ErrorHandler func(*gin.Context, error, int) bool
```

Comment:
What does the `bool` return value signify here? When would you ever return true, to ignore an error?

Reply:
Good question. The `bool` is to tell the wrapper func to continue execution or not. I can't think of a scenario where one would return `true`, but I wanted to provide that flexibility to the developer.

Reply:
I've looked through this a bit, and I think there are several parts here that are somewhat difficult to work around.

First, it's impossible to tell from within the error handler where it's been called from - handling a parameter? Bad request body? You can parse it out of the error string.

I think that for now, we can skip returning that `bool`. When would you ever continue a request when you encountered the kind of error that needs handling? 

Ideally, we use kin-openapi's filter as part of middleware to validate requests - and our own code should never get involved in figuring out why a request is invalid. I've been trying to push as much of this logic out of generated code and into middleware as possible, due to flexibility.